nap/cache/flask_cache.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import unicode_literals
+from __future__ import absolute_import
+try:
+    import flask_caching
+except ImportError as e:
+    raise ImportError("Error loading Flask-Caching: %s" % e)
+
+
+from .base import BaseCacheBackend, DEFAULT_TIMEOUT
```

Comment:
I don't see that you're using DEFAULT_TIMEOUT. It looks like the intent is for it to be passed via kwargs on __init__, so I don't think we need to import it here.